src/displayBackend/wayland/Display.cpp

```diff
@@ -382,6 +382,17 @@ void Display::clearInputCallbacks<TouchCallbacks>(const string& connector)
  * Private
  ******************************************************************************/
 
+void Display::sWaylandLog(const char* fmt, va_list arg)
+{
+	auto len = vsprintf(nullptr, fmt, arg);
```

Comment:
such a waste of resources :(

Reply:
It is only execution time and only if error in wayland.

Reply:
this is why you have my approve